.prettierignore

```diff
@@ -0,0 +1,5 @@
+**/.git
+**/.svn
+**/.hg
```

Comment:
Are these lines necessary?

Reply:
Not here but it's the default ignored files by prettier.
https://prettier.io/docs/en/ignore.html#ignoring-files-prettierignore

By default prettier ignores files in version control systems directories (".git", ".svn" and ".hg") and node_modules (if --with-node-modules CLI option not specified)